app/models/game.rb

```diff
@@ -4,9 +4,15 @@ class Game < ActiveRecord::Base
 
   belongs_to :lobby
 
+  STATUSES = [ :active, :finished ]
+
+  enum status: STATUSES
```

Comment:
I figured enums were a nice addition to make it easy to implement future changes. I don't think they introduce too much complexity and I can always change it later.